knowledge/technical_manual/instructlab/qna.yaml

```diff
@@ -0,0 +1,24 @@
+---
+created_by: makelinux
+domain: AI
+seed_examples:
+  - question: What is InstructLab?
+    answer: InstructLab (Large-scale Alignment for chatBots) is an open source
+      initiative by Red Hat and IBM. It provides a platform for easy engagement
+      with LLM.
+  - question: How users can use InstructLab?
```

Comment:
I think the questions and answers need improvement to be better and more complete english sentences.